bats/tests/helpers/paths.bash

```diff
@@ -108,6 +108,7 @@ if is_windows; then
 fi
 
 PATH_CONFIG_FILE="$PATH_CONFIG/settings.json"
+PATH_SNAPSHOTS_DIR="$PATH_APP_HOME/snapshots"
```

Comment:
I think almost every `PATH_*` points to a directory, we we don't need a `_DIR` suffix. Instead we add `_FILE` when it is ambiguous.

```suggestion
PATH_SNAPSHOTS="$PATH_APP_HOME/snapshots"
```